example-mlem-get-started/code/README.md

```diff
@@ -0,0 +1,39 @@
+# MLEM Get Started
+
+This is an auto-generated repository for use in MLEM
+[Get Started](https://mlem.ai/doc/get-started). It is a step-by-step quick
+introduction into basic MLEM concepts.
+
+The main branch contains only the code needed for the tutorial. Results of each step can be found in tags, and resulting state in correctponding branches.
+
+
+🐛 Please report any issues found in this project here -
+[example-repos-dev](https://github.com/iterative/example-repos-dev).
+
+## Installation
+
+Python 3.7+ is required to run code from this repo.
+
+To be able to commit and push your results please fork and then clone this repo
+
+```console
+$ gh repo fork https://github.com/iterative/example-mlem-get-started
+$ cd example-mlem-get-started
+```
+
+Now let's install the requirements. But before we do that, we **strongly**
+recommend creating a virtual environment with a tool such as
+[virtualenv](https://virtualenv.pypa.io/en/stable/):
+
+```console
+$ python3 -m venv .env
```

Comment:
as far as I know it doesn't install wheels (slow downs and / or makes installing the requirements process slow / broken)

Reply:
It works currently. Do we need to investigate something?

Reply:
no, it's fine (I would not recommend using it, and we don't, but it's simpler and it's not critical). The only thing is - please fix the link that leads to the virtualenv above